### ISSUE TITLE
Exponents in syntax of compound statements needs clarification

### ISSUE THREAD

**Issue by ePaul** [Whiley/WhileyDocs#30]
In the syntax of `DoWhileStmt`, `IfStmt`, `SwitchStmt` and `WhileStmt`, there are raised cursive letters (φ, ω, l, γ) without any explanation on what they mean.

After some confusion I came to the conclusion that they indicate the indentation of the corresponding syntactic elements, but that is not mentioned anywhere (neither in the Indentation section nor in the Block section nor anywhere in the Control Statements section).

**Comment by DavePearce** [comment]
Yeah, that's right --- they refer to indentation.  There needs to be some kind of description of the notation used for expressing the grammar.  I'm going to add more stuff into it as well, like type information.  So, definitely agree something needs to be added.
